lib/ansible/modules/cloud/misc/virt.py

```diff
@@ -23,34 +23,41 @@
 options:
   name:
     description:
-      - name of the guest VM being managed. Note that VM must be previously
-        defined with xml.
+      - Name of the guest VM being managed.
+      - Note that VM must be previously defined with xml.
       - This option is required unless I(command) is C(list_vms).
+    type: str
+    aliases: [ guest ]
   state:
     description:
       - Note that there may be some lag for state requests like C(shutdown)
-        since these refer only to VM states. After starting a guest, it may not
-        be immediately accessible.
-        state and command are mutually exclusive except when command=list_vms. In
-        this case all VMs in specified state will be listed.
-    choices: [ destroyed, paused, running, shutdown ]
+        since these refer only to VM states.
+      - After starting a guest, it may not be immediately accessible.
+      - I(state) and I(command) are mutually exclusive except when C(command=list_vms).
+        In this case all VMs in specified state will be listed.
+    type: str
+    choices: [ destroyed, pause, running, shutdown ]
```

Comment:
Did you mean to change the choice from paused to pause here or is that a typo?

Reply:
The module's argspec actually uses `pause` (for better or worse), so this just fixes the docs to match the module.

Reply:
Indeed, 'paused' would be better, but I don't want to make any impact like that.
Just cleaning up known inconsistencies we ignored in the past.